tests/aws/services/apigateway/test_apigateway_import.py

```diff
@@ -239,7 +239,7 @@ def _is_not_pending():
             try:
                 aws_client.lambda_.delete_function(FunctionName=arn)
             except Exception:
-                LOG.debug(f"Unable to delete function {arn=} in cleanup")
+                LOG.debug("Unable to delete function %s in cleanup", arn)
```

Comment:
This seems to not produce the exact same output. I think `{arn=}` will render to `arn=<arn>` and the change would only output `<arn>`.

Since, it's a debug message it is fine though, I think :)

Reply:
Good point! I missed that, I did all of it manually! I guess it's alright, but good catch, thanks for checking everything 🙇